plugins/modules/sensu_client.py

```diff
@@ -100,10 +100,10 @@
     description:
       - The servicenow definition scope, used to configure the Sensu Enterprise ServiceNow integration (Sensu Enterprise users only).
 notes:
-  - Check mode is supported
-'''
+  - Check mode is supported.
```

Comment:
This doesn't work with suggestions, but can you remove the whole `notes:`? The only note is already covered by `attributes`.

Reply:
Will do this on the next round of review, after the batch commit

Reply:
done